From 72a89b7487a7ffd4ee2d9f9455c7dbd8529ba9ef Mon Sep 17 00:00:00 2001
From: Bai Ping <ping.bai@nxp.com>
Date: Mon, 16 May 2016 17:45:39 +0800
Subject: [PATCH 1064/5242] MLK-12803 ARM: dts: imx: enable snvs poweroff on
 6qdl sabreauto

commit  7c43d1730300c4e7566343f25f804dbe1a330292 from
https://source.codeaurora.org/external/imx/linux-imx.git

Enable snvs poweroff on i.MX6Q/DL sabreauto board.

Signed-off-by: Bai Ping <ping.bai@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6qdl-sabreauto.dtsi |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/boot/dts/imx6qdl-sabreauto.dtsi b/arch/arm/boot/dts/imx6qdl-sabreauto.dtsi
index e63b019..967dabd 100644
--- a/arch/arm/boot/dts/imx6qdl-sabreauto.dtsi
+++ b/arch/arm/boot/dts/imx6qdl-sabreauto.dtsi
@@ -1136,6 +1136,9 @@
 	status = "okay";
 };
 
+&snvs_poweroff {
+	status = "okay";
+};
 
 &ssi2 {
 	assigned-clocks = <&clks IMX6QDL_CLK_SSI2_SEL>;
-- 
1.7.9.5

